integreat_cms/cms/templates/poicategories/poicategory_list.html

```diff
@@ -21,6 +21,14 @@ <h1 class="heading">{% translate "Location Categories" %}</h1>
                             {% endfor %}
                         </div>
                     </th>
+                    <th class="text-sm text-left uppercase py-3 px-2"
+                        title="{% translate "Icon" %}">{% translate "Icon" %}</th>
+                    <th class="text-sm text-right uppercase py-3 pr-2"
+                        title="{% translate "Color code" %}">
+                        {% translate "Color code" %}
+                    </th>
+                    <th class="text-sm text-right uppercase py-3 pr-2"
```

Comment:
```suggestion
                    <th class="text-sm text-left uppercase py-3 pr-2"
```
I think we can combine color name and preview into one column.